src/compiler/error.h

```diff
@@ -0,0 +1,45 @@
+#pragma once

```

Comment:
Since this is used by the VM, should this live somewhere else? eg. `src/ds/`

Reply:
Good point